### DISCUSSION TITLE
How do I provide Postgres ssl mode

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2997] [Archive]
Currently, I see no option to provide Postgres SSL mode env, is there any?

**Comment by rijkvanzanten** [comment]
Any `DB_*` env var is passed on to the Knex instance, so you can set `DB_SSL=true` 🙂

**Comment by ghost** [comment]
@rijkvanzanten How about `{ ssl: { rejectUnauthorized: false  }} `

**Reply by rijkvanzanten** [reply]
```
DB_SSL__REJECT_UNAUTHORIZED
```

👍🏻

**Reply by rijkvanzanten** [reply]
Make sure not to combine `DB_SSL` and `DB_SSL__*` though. The pg driver requires one or the other
